src/rpc/blockchain.cpp

```diff
@@ -1004,6 +1004,13 @@ static UniValue pruneblockchain(const JSONRPCRequest& request)
 
     LOCK(cs_main);
 
+    // reject manual pruning in case if a blockfilterindex is syncing
+    ForEachBlockFilterIndex([](BlockFilterIndex& index) {
```

Comment:
bd11606a95022478e6ef1ccaada1589cff13a327

Can be checked before locking `cs_main`?

Reply:
Fixed.